doc/app-bar-top.md

```diff
@@ -10,9 +10,18 @@ This component can help you to show some infos and/or actions on the top of the
 Every time you want to configure an quick acess to some basic infos or actions
 
 ### How to use it?
-If you need to configure just a basic App Bar, with title and some simple actions, just configure at your view the base theme. You can check how to do it [here](getting-started.md).
+Add the app bar component in your xml layout file
+Important! Do not use the default theme, use the NoActionBar variant. You can check more infos [here](getting-started.md).
```

Comment:
Não é boa prática usar link com texto "here" ou "aqui". Idealmente, deveríamos usar algo como:
`You can check more info in our [Getting Started guide](getting-started.md)`

Reply:
Fixed